### ISSUE TITLE
Fix string keys that are inconsistent with object-style access

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#222]
In https://github.com/phetsims/chipper/issues/832#issuecomment-583877692 it was reported that there are a few strings in string files that are named such that key names and object names would overlap.  The list is available in the issue comment linked above, but I'll repeat it here for easy reference:

```
AREA_MODEL_COMMON levelPrompt.oneProduct / levelPrompt.oneProduct.totalArea
GAS_PROPERTIES holdConstant / holdConstant.nothing
MOLARITY molarity / molarity.title
VEGAS label.score / label.score.max
WAVE_INTERFERENCE screen / screen.waves (+3)
```

These string keys should be changed so that this problem is avoided.

**Comment by jbphet** [comment]
The only common-code string in the list above is `VEGAS/label.score`.  I just when through the code base and determined which sims will need a maintenance release when this string is updated, here is the list:

+ area-builder
+ area-model-algebra
+ area-model-multiplication
+ arithmetic
+ balancing-act
+ balancing-chemical-equations
+ build-an-atom
+ fraction-matcher
+ fractions-equality
+ graphing-lines
+ reactants-products-and-leftovers

**Comment by jbphet** [comment]
The general proposal for how to handle these is to make rename the key that looks like it has sub-values to <key-name>.text, for example, `GAS_PROPERTIES/holdConstant` will be changed to `GAS_PROPERTIES/holdConstant.text`.  This seems like a reasonable approach, and I'm going to give it a try.

**Comment by jbphet** [comment]
I've worked through the checklist for the GAS_PROPERTIES string, here is what was done.  Note that a maintenance release was also needed for Gases Intro.


- [ ] For later verification, make a record of the string key that is changing and its value for at least one
translation (bonus points for doing multiple translations)
- [ ] Determine all places where the string is used.  If it is used in another sim, you'll need to do a maintenance
release.
- [ ] Put Rosetta into maintenance mode to avoid any translation submissions while this is in progress (see the Admin
Guide for information on how to do this).
- [ ] Change the value of the string key on master.  This will need to be done in the string file and at all usage
sites.
- [ ] Build the sim and verify that there are no string errors, then test the built sim and verify the string looks
correct.
- [ ] Change the value of the string key on the release branch.  Again his will need to be done in the string file and
at all usage sites.
- [ ] Build the sim and verify that there are no string errors, then test the built sim and verify the string looks
correct.
- [ ] Change the value of the string key in all translations on the master branch in babel.
- [ ] Use the `locale=x` to test a couple of locales for which translations have been submitted and make sure the string
looks correct.
- [ ] Create a new RC on the release branch.
- [ ] Check if there are any unverified changes on this branch.  If there are, make sure that the appropriate amount of
testing is performed on the RC before publishing (the details of what is "appropriate" is beyond the scope of this
document).  If there are no such changes, it is generally fine to move forward with publication without verification.
- [ ] Publish the simulation.
- [ ] If the string is used by another simulation, you'll need to update the version of the repo where the change
occurred that is used by the sim.  Usually this means creating or updating a branch of the repo where the change was
made and then updating the dependencies list for the dependant simulation.  You'll need to do an RC and production
release for this sim as well.
- [ ] Take Rosetta out of maintenance mode (again, see the Admin Guide for information on how to do this).
- [ ] Verify that the string key change worked by going to Rosetta (https://phet.colorado.edu/translate/), selecting the
simulation for which the update or updates were made, choosing a locale for which a translation exists, pressing "Go",
and verifying that the old string key is gone, the new one is shown, and the value of the new one is correct.  Do this
for all locales for which values were recorded above.

**Comment by jonathanolson** [comment]
RCs (minus balancing-act):

area-builder 1.1 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [area-builder 1.1.20-rc.1](https://phet-dev.colorado.edu/html/area-builder/1.1.20-rc.1/area-builder_en.html)

area-model-algebra 1.2 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [area-model-algebra 1.2.1-rc.1](https://phet-dev.colorado.edu/html/area-model-algebra/1.2.1-rc.1/phet/area-model-algebra_en_phet.html)

area-model-decimals 1.2 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [area-model-decimals 1.2.1-rc.1](https://phet-dev.colorado.edu/html/area-model-decimals/1.2.1-rc.1/phet/area-model-decimals_en_phet.html)

area-model-introduction 1.2 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [area-model-introduction 1.2.1-rc.1](https://phet-dev.colorado.edu/html/area-model-introduction/1.2.1-rc.1/phet/area-model-introduction_en_phet.html)

area-model-multiplication 1.2 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [area-model-multiplication 1.2.1-rc.1](https://phet-dev.colorado.edu/html/area-model-multiplication/1.2.1-rc.1/phet/area-model-multiplication_en_phet.html)

arithmetic 1.0 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [arithmetic 1.0.24-rc.1](https://phet-dev.colorado.edu/html/arithmetic/1.0.24-rc.1/arithmetic_en.html)

balancing-chemical-equations 1.2 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [balancing-chemical-equations 1.2.10-rc.1](https://phet-dev.colorado.edu/html/balancing-chemical-equations/1.2.10-rc.1/phet/balancing-chemical-equations_en_phet.html)

build-a-fraction 1.0 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [build-a-fraction 1.0.12-rc.1](https://phet-dev.colorado.edu/html/build-a-fraction/1.0.12-rc.1/phet/build-a-fraction_en_phet.html)

build-an-atom 1.6 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [build-an-atom 1.6.14-rc.1](https://phet-dev.colorado.edu/html/build-an-atom/1.6.14-rc.1/build-an-atom_en.html)

fraction-matcher 1.2 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [fraction-matcher 1.2.1-rc.1](https://phet-dev.colorado.edu/html/fraction-matcher/1.2.1-rc.1/phet/fraction-matcher_en_phet.html)

fractions-equality 1.1 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [fractions-equality 1.1.1-rc.1](https://phet-dev.colorado.edu/html/fractions-equality/1.1.1-rc.1/phet/fractions-equality_en_phet.html)

fractions-intro 1.0 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [fractions-intro 1.0.12-rc.1](https://phet-dev.colorado.edu/html/fractions-intro/1.0.12-rc.1/phet/fractions-intro_en_phet.html)

fractions-mixed-numbers 1.0 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [fractions-mixed-numbers 1.0.12-rc.1](https://phet-dev.colorado.edu/html/fractions-mixed-numbers/1.0.12-rc.1/phet/fractions-mixed-numbers_en_phet.html)

graphing-lines 1.3 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [graphing-lines 1.3.10-rc.1](https://phet-dev.colorado.edu/html/graphing-lines/1.3.10-rc.1/phet/graphing-lines_en_phet.html)

reactants-products-and-leftovers 1.2 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [reactants-products-and-leftovers 1.2.11-rc.1](https://phet-dev.colorado.edu/html/reactants-products-and-leftovers/1.2.11-rc.1/phet/reactants-products-and-leftovers_en_phet.html)

waves-intro 1.0 (https://github.com/phetsims/rosetta/issues/222)

- [ ] [waves-intro 1.0.2-rc.1](https://phet-dev.colorado.edu/html/waves-intro/1.0.2-rc.1/phet/waves-intro_en_phet.html)

**Comment by jbphet** [comment]
I verified that the changes and redeployments worked correctly by loading each of the affected simulations in Rosetta with `de` (German) selected as the destination locale, and then verifying that the new string key is visible and the old one is not, and the value for the German translation appears to be correct.  Below is a screenshot of one such case - one of the area model sims where the key `levelPrompt.oneProduct` was changed to `levelPrompt.oneProduct.text`.

![image](https://user-images.githubusercontent.com/3082831/75287093-ff88e180-57d6-11ea-8be5-68e360bef73e.png)

I believe our work here is done.  Closing.
